app/modules/asset_groups/models.py

```diff
@@ -558,7 +555,7 @@ class AssetGroup(db.Model, HoustonModel):
 
     description = db.Column(db.String(length=255), nullable=True)
 
-    config = db.Column(db.JSON, nullable=True)
+    config = db.Column(db.JSON, default=lambda: {}, nullable=False)
```

Comment:
Leaving this potentially nullable was causing issues. Safer to have it as an empty dict.